test/test.js

```diff
@@ -4,38 +4,68 @@ var test = require('tape');
 var load = require('load-json-file');
 var isoBands = require('../dist/marchingsquares.js').isoBands;
 var isoLines = require('../dist/marchingsquares.js').isoLines;
+var quadTree = require('../dist/marchingsquares.js').quadTree;
```

Comment:
@RaumZeit a side note: is convention to [capitalize constructors](https://stackoverflow.com/a/1564489/4530144), so `quadTree` should be `QuadTree`

Reply:
Thanks for the hint! I already changed the sources in the `master` branch accordingly :)